ceph/ceph_admin/orch.py

```diff
@@ -104,11 +104,17 @@ def check_service_exists(
                 f"{running}/{count} {service_name if service_name else service_type} up... retrying"
             )
 
-            _retries = 3 if (_count != count and running != count) else _retries - 1
-            _count = count
+            if count + running < 1:
+                continue
```

Comment:
We could leave this as `if count + running == 0: continue`

Reply:
I think Its good to validate positive number.

Reply:
I don't think `count + running` is going to `<= 0`... it is not important... but I thought it would avoid a confusion.